chromium_src/chrome/browser/safe_browsing/BUILD.gn

```diff
@@ -15,6 +15,10 @@ source_set("install_verification") {
     "//chrome/browser/install_verification/win/module_verification_common.h",
     "//electron/chromium_src/chrome/browser/install_verification/win/module_ids.cc",
   ]
+
+  public_deps = [
```

Comment:
does this need to be public?

Reply:
yeah, deps seems sufficient and I still couldn't reproduce this per discussion on Slack